js/siteorigin-panels/view/builder.js

```diff
@@ -689,7 +689,7 @@ module.exports = Backbone.View.extend( {
 	 */
 	updateEditorContent: function ( content ) {
 		// Switch back to the standard editor
-		if ( this.config.editorType !== 'tinymce' || _.isUndefined( tinyMCE ) || _.isNull( tinyMCE.get( "content" ) ) ) {
+		if ( this.config.editorType !== 'tinymce' || typeof tinymce === 'undefined' || _.isNull( tinyMCE.get( "content" ) ) ) {
```

Comment:
They are, which explains why I didn't pick on this.

I'll switch them over to TinyMCE now.